template/compose.tpl/containers_network.yml.tpl

```diff
@@ -52,7 +42,7 @@ services:
     networks:
     - tyk
   candig_server:
-    image: c3genomics/candig_server
+    build: ../../candig-server
```

Comment:
I think we should not publish a hardcode the build step ;)

Reply:
Yea sorry totally forget this. Will remove.